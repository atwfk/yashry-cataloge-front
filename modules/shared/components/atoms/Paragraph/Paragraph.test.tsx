import React from "react";
import P from "./Paragraph";
import renderer from "react-test-renderer";

describe("Paragraph", () => {
  it("should render Paragraph with small text with primary color", () => {
    const tree = renderer
      .create(<P fontSize="sm" fontColor="primary" fontWeight="normal" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("should render Paragraph with medium text with accent color", () => {
    const tree = renderer
      .create(<P fontSize="md" fontColor="accent" fontWeight="medium" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it("should render Paragraph with large text with error color", () => {
    const tree = renderer
      .create(<P fontSize="lg" fontColor="error" fontWeight="bold" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
