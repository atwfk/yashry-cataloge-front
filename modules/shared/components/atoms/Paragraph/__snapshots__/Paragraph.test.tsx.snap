// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should render Paragraph with large text with error color 1`] = `
<p
  className="text-lg text-error font-bold"
/>
`;

exports[`Paragraph should render Paragraph with medium text with accent color 1`] = `
<p
  className="text-base text-accent font-medium"
/>
`;

exports[`Paragraph should render Paragraph with small text with primary color 1`] = `
<p
  className="text-sm text-primary font-normal"
/>
`;
